src/assessments/cognitive/assessment.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RedundantEntry } from 'assessments/cognitive/test-steps/redundant-entry';
+import { VisualizationType } from 'common/types/visualization-type';
+import { test as content } from 'content/test';
+import * as React from 'react';
+import { AssessmentBuilder } from '../assessment-builder';
+import { Assessment } from '../types/iassessment';
+import { Authentication } from './test-steps/authentication';
+
+const { guidance } = content.cognitive;
+
+const key = 'cognitive';
+const title = 'Cognitive';
+
+const gettingStarted: JSX.Element = (
+    <React.Fragment>
+        <p>
+            When interfaces require repetitive entry of user data, or necessitate that users recall
```

Comment:
Reading this out loud, I'd probably drop this comma
```suggestion
            When interfaces require repetitive entry of user data or necessitate that users recall
```